### PR TITLE
Closes #53

### LINKED ISSUE #53: Enable setting Matlab_Root to point to MATLAB location
If the environment variable Matlab_Root is set, it is currently being ignored. It would be good to support this variable, instead of always relying on cmake to locate MATLAB.

### PR DESCRIPTION
**Pull request by duncanpo**
Change a cmake_policy to enable setting Matlab_Root to point to MATLAB location

### CODE REVIEW

File: libmexclass/cpp/CMakeLists.txt [hunk 1]
```diff
@@ -1,5 +1,8 @@
 cmake_minimum_required(VERSION 3.0.0)
 
+# Set policy to allow setting Matlab_ROOT
```
Comment (kevingurney) [inline libmexclass/cpp/CMakeLists.txt]: Suggested wording: "Workaround to support specifying Matlab_ROOT to find_package(Matlab)"

**Comment by kevingurney** [comment]
Thanks for the pull request, @duncanpo ! Overall this change looks good to me.

Can you please create a GitHub issue to track this Enhancement and [associate this pull request with it](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)? You can do this by writing something like "Closes #\<issue-id\>" in the pull request description or title.

Thank you!

File: libmexclass/cpp/CMakeLists.txt (hunk 1 above)
Comment (duncanpo) [reply libmexclass/cpp/CMakeLists.txt]: I don't really think this is a workaround. Can we drop that word? "Support specifying Matlab_Root to find_package(Matlab)"

Comment (kevingurney) [reply libmexclass/cpp/CMakeLists.txt]: Sorry for the confusion. The [`FindMatlab`](https://cmake.org/cmake/help/latest/module/FindMatlab.html) documentation doesn't mention the need to set this policy anywhere, which is why I thought this was a workaround.

After thinking more about this, my understanding is that this CMake policy must be set explicitly by *any* client project that wants to use `find_package` with a `<PACKAGE>_ROOT` variable.

Maybe we could write something like the following in the comment instead?

"CMake policy `CM0074` must be set explicitly by any client project that wants to use `find_package` with a `<PACKAGE>_ROOT` variable. Therefore, we set this policy to enable support for the `Matlab_ROOT` flag  with `find_package(Matlab)`."

Comment (duncanpo) [reply libmexclass/cpp/CMakeLists.txt]: Agree with your latest suggestion. I will make the change.

Comment (kevingurney) [reply libmexclass/cpp/CMakeLists.txt]: Thanks!

**Review by kevingurney — APPROVED** [review]

**Pull request merged**